src/test/java/teammates/test/cases/browsertests/AdminAccountDetailsPageUiTest.java

```diff
@@ -11,7 +11,7 @@
 import teammates.test.pageobjects.AdminAccountDetailsPage;
 
 /**
- * SUT: {@link Const.ActionURIs#ADMIN_ACCOUNT_DETAILS_PAGE}.
+ * SUT: {@link Const.WebPageURIs#ADMIN_ACCOUNTS_PAGE}.
```

Comment:
I don't think you understanding the full context of the migration process yet. Therefore, my suggestion is that leave all `*UiTest`'s `SUT` documentation there and don't touch them.